cmd/containerd/command/main.go

```diff
@@ -130,6 +132,19 @@ can be used and modified as necessary as a custom configuration.`
 			return err
 		}
 
+		// Initialize openTelemetry tracing
+		shutdown, err := tracing.InitOpenTelemetry(config)
+		log.G(ctx).Debug("Initializing opentelemetry")
+		if err != nil {
+			errors.Wrap(err, "failed to initialize openTelemetry tracing")
+		}
+		defer shutdown()
+
+		// Get tracer
+		ctrdTracer := otel.Tracer("containerd")
+		ctx, mainCtrdSpan := ctrdTracer.Start(gocontext.Background(), "containerd-expoerter")
+		defer mainCtrdSpan.End()
```

Comment:
@Aneurysm9 @alakesh Is it common to have the "main" span?

Reply:
It probably depends largely on your program's usage pattern.  If you expect a `main()` invocation to be quick and represent a single unit of work, then it might make sense to have one.  On the other hand, if your program is long-lived then the value of having such a span decreases and you'll want to ensure that spans representing individual units of work are linked to the main span but create their own traces.